lib/jekyll-github-metadata/repository.rb

```diff
@@ -98,6 +98,10 @@ def releases
         memoize_value :@releases, Value.new(proc { |c| c.releases(nwo) })
       end
 
+      def commits
+        memoize_value :@commits, Value.new(proc { |c| c.commits(nwo) })
```

Comment:
We'd have to turn it off for the client while this call is made. Something like: https://github.com/octokit/octokit.rb/issues/554#issuecomment-72337218